backend/routes/calendar.py

```
@@ -114,14 +114,17 @@ def save_assignments(body: SaveAssignmentsBody):
-    )
-    return {"assignments": rows}
+    try:
+        today = datetime.utcnow().strftime("%Y-%m-%d")
+        rows = table("assignments").select(
+            "*",
+            filters={"user_id": f"eq.{user_id}", "due_date": f"gte.{today}"},
+            order="due_date.asc",
+            limit=20,
+        )
+        return {"assignments": rows or []}
+    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)